rust/noosphere/src/noosphere.rs

```diff
@@ -50,7 +50,7 @@ pub enum NoosphereNetwork {
     /// Uses an HTTP REST client to interact with various network resources
     Http {
         gateway_api: Option<Url>,
-        ipfs_api: Option<Url>,
+        ipfs_gateway_url: Option<Url>,
```

Comment:
I think there may have been some confusion in terminology here.

IPFS "gateway" is generally used to refer to the HTTP gateway, but here we literally mean the RPC which is a REST API.

Reply:
Actually, nevermind. This seems correct.